templates.go

```diff
@@ -658,70 +658,26 @@ write_files:
   permissions: 0544
   content: |
       #!/bin/bash
-      while ! curl --output /dev/null --silent --head --fail --cacert /etc/kubernetes/ssl/apiserver-ca.pem --cert /etc/kubernetes/ssl/apiserver-crt.pem --key /etc/kubernetes/ssl/apiserver-key.pem "https://{{.Cluster.Kubernetes.API.Domain}}:443"; do sleep 1 && echo 'Waiting for master'; done
-      curl -o /opt/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.6.1/bin/linux/amd64/kubectl 
+      # get kubectl
+      curl -o /opt/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.6.1/bin/linux/amd64/kubectl
```

Comment:
This should actually be configured via the clustertpr. See https://github.com/giantswarm/kubernetesd/blob/a8528d72f2a2ff7b37c5d6674684a03c543830c1/kubernetes/configmap.yaml#L90. I realize now that this is not specific to KVM so the config should be moved in kubernetesd. Anyway, at the moment here the URL should be replaced by something like this. 

```
curl -o /opt/bin/kubectl {{ .Cluster.KVM.Kubectl.GoogleAPI.URL }}
```

Lets please create an issue for this and address it in the next steps.